### ISSUE TITLE
codehilite: allow passing 'cssclass' to pygments

### ISSUE THREAD

**Issue by lahwaacz** [Python-Markdown/markdown#286]
The users should not be limited to the default `codehilite` CSS class, please allow manually setting the `cssclass` arguments for pygments. For example:

```
html = markdown.markdown(text, extensions=['codehilite(cssclass="codeblock")'])
```

**Comment by waylan** [comment]
This is already [possible](https://github.com/waylan/Python-Markdown/blob/master/markdown/extensions/codehilite.py#L234), although admittedly not documented outside of comments in the code. In fact, it looks like a few of CodeHilite's config options are not documented, A Pull Request with a documentation patch is certainly welcome.

**Comment by lahwaacz** [comment]
Thanks, in return I'm working on improving the documentation (also for other extensions).

Interestingly I found that it's implemented in c908b046, but the [html documentation](http://pythonhosted.org/Markdown/extensions/code_hilite.html#usage) is not updated.

**Comment by waylan** [comment]
Yeah, unfortunately, the extensions are not as well documented as the main library. If volunteers each picked a few they cared about and provided pull requests, I would be happy to merge them in. Note, however, that I anticipate doing a release this weekend, so if you want your additions in do it now.

**Comment by lahwaacz** [comment]
The pull request is at #288, please review. Note that Footnotes/UNIQUE_IDS option is still undocumented. I'll try to finish my work today so that you can do a release.

**Comment by lahwaacz** [comment]
That's it -- I've just finished! Hope you'll like it...

**Comment by waylan** [comment]
Closing this as the documentation issue has been addressed with #288.
